pkg/consensus/recovery_message_test.go

```diff
@@ -114,23 +114,6 @@ func TestRecoveryMessage_Setters(t *testing.T) {
 	})
 }
 
-//TODO NEO3.0: Update binary
-/*
-func TestRecoveryMessage_DecodeFromTestnet(t *testing.T) {
```

Comment:
This one was for recovery and it'd be nice to add it also.

Reply:
Fixed. This was a message from a real test net for a specific bug (invalid prepare request format inside recovery). Added a recovery messasge dump with the same contents.